go.mod

```diff
@@ -6,10 +6,16 @@ replace github.com/creasty/defaults => github.com/signalfx/defaults v1.2.2-0.201
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 
+replace k8s.io/api => k8s.io/api v0.0.0-20181110191121-a33c8200050f
```

Comment:
This isn't necessary.  Just do `go get k8s.io/api@a33c8200050f` and it will set the version in the `require` section below.

Reply:
I tried it but it did not work for me. Was seeing this and then gave up :)

```
build github.com/signalfx/signalfx-agent/cmd/monitorcodegen: cannot load k8s.io/api/admissionregistration/v1alpha1: module k8s.io/api@latest found (v0.17.0), but does not contain package k8s.io/api/admissionregistration/v1alpha1
```

Reply:
Also I just realized you are downgrading this package which is generally the opposite direction we want to go.  Let me check and see if I can get it building without this.

Reply:
Thanks